src/modinfodialogconflicts.h

```diff
@@ -51,6 +51,39 @@ class GeneralConflictsTab : public QObject
   FilterWidget m_filterOverwritten;
   FilterWidget m_filterNoConflicts;
 
+  struct GeneralConflictNumbers
+  {
+    int numTotalFiles = 0;
+    int numTotalLoose = 0;
+    int numTotalArchive = 0;
+    int numNonConflicting = 0;
+    int numNonConflictingLoose = 0;
+    int numNonConflictingArchive = 0;
+    int numOverwrite = 0;
+    int numOverwriteLoose = 0;
+    int numOverwriteArchive = 0;
+    int numOverwritten = 0;
+    int numOverwrittenLoose = 0;
+    int numOverwrittenArchive = 0;
+
+    void clear() {
```

Comment:
`*this = {}`?

Reply:
Maybe not provide `clear` and simply re-assign a new object in the actual code? There is nothing to gain from a custom `clear` since this is simply a bunch of `int`s.